src/daemon/org.projectatomic.rpmostree.xml

```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE node PUBLIC
+"-//freedesktop//DTD D-BUS Object Introspection 1.0//EN"
+"http://www.freedesktop.org/standards/dbus/1.0/introspect.dtd">
+<node name="/" xmlns:doc="http://www.freedesktop.org/dbus/1.0/doc.dtd">
+
+  <interface name="org.projectatomic.rpmostree.Manager">
+    <property name="DefaultSysroot" type="o" access="read"/>
+
+    <!--
+         Returns a object path for the Sysroot
+         at the give path. If the path is not
+         not a valid sysroot an error is returned.
+    -->
+    <method name="GetSysroot">
+      <arg name="path" type="s" direction="in"/>
+
+      <arg name="sysroot" type="o" direction="out"/>
+    </method>
```

Comment:
I was trying to keep parity with the command line, if the command line is going to use the dbus api to run it's commands and continue to support that option then this needs to support it as well.

On the other hand, we could drop that option from the command line for the sub commands that use dbus or decide that if that option is provided it talks to libostree directly, though i wouldn't be thrilled about having 2 code paths. I'm happy to defer to whatever you think is better.

As a side note, i've been using this to run rpm-ostree from a SPC, but that's neither here nor there for what we want long term.